haskell/src/Stablecoin/Client/Impl.hs

```diff
@@ -261,14 +338,27 @@ getStorage (arg #contract -> contract) = do
     Right storage -> pure storage
     Left err -> throwM $ SCEExpressionParseError storageExpr err
 
+-- | Safer version of 'Client.readBigMapValue' that accepts a typed `BigMapId`.
+safeReadBigMapValue
+  :: (NicePackedValue k, NiceUnpackedValue v, UnpackedValScope (ToT v))
+  => BigMapId k v -> k -> MorleyClientM v
+safeReadBigMapValue (BigMapId bigMapId) k = Client.readBigMapValue bigMapId k
+
+-- | Safer version of 'Client.readBigMapValueMaybe' that accepts a typed `BigMapId`.
+safeReadBigMapValueMaybe
```

Comment:
Wouldn't it be better to put it into `morley` repository (along with typed `BigMapId`)?

Reply:
Done